src/lib/configuration_objects.py

```diff
@@ -37,3 +37,8 @@ def __init__(self, raw_field, dataset_name, coding_configurations, raw_field_fol
         if id_field is None:
             id_field = "{}_id".format(self.raw_field)
         self.id_field = id_field
+
+class PipelineEvents(object):
+    PIPELINE_EVENT = {"1":"CodaAdd", "2":"FetchingRawData", "3":"GeneratingOutputs", "4":"CodaGet",
```

Comment:
Can you make this look more like an enum rather than a dict? See e.g. SomaliaCodes in Core for a full example, but basically something like:
```
class PipelineEvents(object):
    CODA_ADD = "CodaAdd"
    FETCHING_RAW_DATA = "FetchingRawData"
    ...

    CHOICES = [PipelineEvents.CODA_ADD, ...]
```